### DISCUSSION TITLE
Welcome to RadioLib Discussions!

### DISCUSSION THREAD

**Discussion by jgromes** [jgromes/RadioLib#209] [General]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:
* Ask questions you’re wondering about or if you're not sure it's worth creating an issue.
* Share ideas.
* Engage with other community members.
* Welcome others and be open-minded.

If you have a question in the Q&A section, please mark it as answered once it's resolved. It helps to navigate the threads, as well as to sort the finished and on-going discussions.

**Comment by sluissantos** [comment]
Hii,

Is it possible to port this application to Zephyr? What challenges and technical considerations would be involved in the process?

I need to do a LoRA 2.4GHz application, using SX1280 transceiver.

**Reply by jgromes** [reply]
Oops, looks like I forgot to lock these generic posts. @sluissantos I moved your question to a new discussion ;)
